device-types/Delta/DPST-2130CB-A.yaml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
These dont have any power or interfaces on them. What is the point of this device? Should this actually be a PSU module?

Reply:
converted power supplies to modules